### PR TITLE
[IMP] theme_*: adapt themes with new `s_unveil` snippet

### PR DESCRIPTION
**Pull request by xlu-odoo**
*: anelusia, avantgarde, aviato, beauty, bewise, bookstore, ernark, graphene, kea, kiddo, loftspace, monglia, nano, notes, odoo experts, orchid, paptic, real_estate, treehouse, vehicle, yes, zap

This commit adapts the design of new s_unveil snippet for multiple themes.

task-4104471
part-of: task-4077427

requires: https://github.com/odoo/odoo/pull/176167

### CODE REVIEW

File: theme_beauty/views/snippets/s_unveil.xml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+<odoo>
+    
+<template id="s_unveil" inherit_id="website.s_unveil">
+    <!-- Section -->
+    <xpath expr="//section" position="attributes">
+        <attribute name="class" add="o_cc o_cc4" separator=" "/>
+    </xpath>
+    <!-- Title -->
+    <xpath expr="//h2" position="replace" mode="inner">
+        Unleash Your Beauty
```
Comment (anso-odoo) [inline theme_beauty/views/snippets/s_unveil.xml:11]: Not sure about this one 😂 otherwise LGTM !

**Review by anso-odoo — APPROVED** [review]
Good job 👏

File: theme_clean/views/snippets/s_unveil.xml [hunk 2]
```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (sobo-odoo) [inline theme_clean/views/snippets/s_unveil.xml:1]: You forgot to add this one in the manifest :wink:

**Review by sobo-odoo — CHANGES_REQUESTED** [review]
Here is my review :slightly_smiling_face:

**Comment by sobo-odoo** [comment]
@xlu-odoo (just in case you didn't see, I posted my review for design-themes :arrow_up: :wink: )

**Comment by xlu-odoo** [comment]
> @xlu-odoo (just in case you didn't see, I posted my review for design-themes ⬆️ 😉 )

Yes, I did see it and thought that I pushed my changes.  😶‍🌫️

**Review by sobo-odoo — APPROVED** [review]
@robodoo r+

**Pull request closed without merging**